content/1.md

```diff
@@ -1,8 +1,103 @@
+- 26:50 [Old Reddit Redirect][old-reddit-redirect]
+- 27:48 [Unhook &ndash; Remove YouTube Recommended Videos][unhook]
+- 28:33 [Imagus][imagus]
+
+## Muita softia
+
+- 29:26 [TouchCursor][touchcursor]
+- (31:39 [Ultimate Hacking Keyboard (UHK)][uhk])
+- 33:49 [KeyMouse][keymouse]
+- 36:10 [AutoHotkey (AHK)][ahk]
+  ja [AutoHotkey v2][ahk-v2]
+  sekä [Matiaksen koleat AutoHotkey-skriptit][ahk-scripts]
+- 41:10 [Lightshot][lightshot]
+- 41:52 [ShareX][sharex]
+  ja [Greenshot][greenshot]
+- 42:52 [Everything][everything]
+- 45:01 [Windows Task Scheduler][task-scheduler] (natiivi softa)
+- 45:14 [Hanneksen brutaali Auto shutdown -skripti][auto-shutdown]
+- 45:40 [Dorg: Hanneksen Downloads organizer -skripti][downloads-organizer]
+- 46:14 [Windows Clipboard][clipboard] (natiivi leikepöytämanageri)
+- 47:11 [TaskbarX][taskbarx]
+- 48:32 [f.lux][f.lux]
+  ja [Windows Night Light][night-light]
+- 49:27 [Windows Terminal][windows-terminal]
+
+## Jakson valinnat
+
+- 50:19 Jakson valinnat
+- 50:41 Matias: [Tailwind JIT][tailwind-jit]
+- 55:42 Hannes: [Androidin Display size -asetus][android-display-scaling]
+  - Jaksossa kutsuttiin nimellä "Display scaling"
+
+[ahk-scripts]: https://github.com/mtsknn/AutoHotkey
+[ahk-v2]: https://www.autohotkey.com/v2/
+[ahk]: https://www.autohotkey.com/
+[android-display-scaling]: https://support.google.com/accessibility/android/answer/6006972
+[auto-shutdown]: https://github.com/Steellow/auto_shutdown.bat
+[clipboard]: https://support.microsoft.com/en-us/windows/clipboard-in-windows-10-c436501e-985d-1c8d-97ea-fe46ddf338c6
+[downloads-organizer]: https://github.com/Steellow/dorg_windows
+[everything]: https://www.voidtools.com/
+[f.lux]: https://justgetflux.com/
+[github-settings]: https://github.com/settings/appearance
+[greenshot]: https://getgreenshot.org/
+[imagus]: https://addons.mozilla.org/en-US/firefox/addon/imagus/
+[keymouse]: https://github.com/iscooool/KeyMouse
+[leechblock]: https://addons.mozilla.org/en-US/firefox/addon/leechblock-ng/
+[lightshot]: https://app.prntscr.com/
+[monaco-markdown-github-editor]: https://addons.mozilla.org/en-US/firefox/addon/monaco-markdown-github-editor/
+[night-light]: https://support.microsoft.com/en-us/windows/set-your-display-for-night-time-in-windows-10-18fe903a-e0a1-8326-4c68-fd23d7aaf136
+[old-reddit-redirect]: https://addons.mozilla.org/en-US/firefox/addon/old-reddit-redirect/
+[playback-speed]: https://addons.mozilla.org/en-US/firefox/addon/playback-speed/
+[powertoys]: https://github.com/microsoft/PowerToys
+[refined-github]: https://addons.mozilla.org/en-US/firefox/addon/refined-github-/
+[res]: https://addons.mozilla.org/en-US/firefox/addon/reddit-enhancement-suite/
+[sharex]: https://getsharex.com/
+[stylus]: https://addons.mozilla.org/en-US/firefox/addon/styl-us/
+[tailwind-jit]: https://blog.tailwindcss.com/just-in-time-the-next-generation-of-tailwind-css
+[task-scheduler]: https://en.wikipedia.org/wiki/Windows_Task_Scheduler
```

Comment:
@Steellow Can you find a better URL for this? 😅

Reply:
I think the setting location varies on different android phones. Anyway, didn't find better url so that's good 👍🏽

Reply:
I think you are referring to the `android-display-scaling` URL. All right, nice. 👍

I was asking about the `task-scheduler` URL. Do you think it's good enough?

Reply:
That one or maybe this: https://www.howtogeek.com/123393/how-to-automatically-run-programs-and-set-reminders-with-the-windows-task-scheduler/ ?

Reply:
Yea that's much better, thanks 👍